python/README.md

```diff
@@ -40,3 +40,27 @@ docker run -t -i --rm=true -e "PYTHONPATH=/manager/python/:/manager/client/rhel/
 ```
 
 Within the container shell, just run  `pytest /manager/python/test/` to run all tests.
+
+# Linting
+
+Uyuni uses [`pylint`](https://pylint.pycqa.org/en/latest/index.html) and [`black`](https://black.readthedocs.io/en/stable/) for enforcing and automatic application code linting and formatting.
+
+For pylint, Uyuni uses Google's [pylintrc](https://google.github.io/styleguide/pylintrc) configuration with the following modifications:
+
+* Use 4 spaces instead of 2 for indentation.
+* Line length rule is disabled, because Black enforces line length of `88` characters per line (where possible).
+
+Use the provided script to run `pylint` and `black` over a set of files:
```

Comment:
Could we maybe introduce `pre-commit` so this is triggered automatically doing `git commit`?

Reply:
I'm one of those who don't use [`pre-commit`](https://pre-commit.com/) automatically. For me it would be okay to have it available, but I don't really see the advantage over a script that runs `lint.sh` with the changed files as a [Git pre-commit hook ](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks).

Either way, I think we can add that later on.

Reply:
Sure, we can go with this later in a separeted PR. I'm kind of used to `pre-commit`, maybe not enforcing it, but of course, we can evaluate other alternatives, we will see.